sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -303,22 +322,80 @@ case class GpuParquetMultiFilePartitionReaderFactory(
     throw new IllegalStateException("GPU column parser called to read rows")
   }
 
+  private def resolveURI(path: String): URI = {
+    try {
+      val uri = new URI(path)
+      if (uri.getScheme() != null) {
```

Comment:
Do we need to worry about Hadoop's default fs setting here?  If the path doesn't have a scheme on it, I'm not sure we can assume that means local file.

Reply:
I thought since it comes from a Hadoop Path it always had the scheme on it. I know in my testing on our hdfs cluster it always added the scheme even when it was left off.  I'll dig in a bit more to verify.

Reply:
I should clarify that based on the way its based through to use and its a Path and we use the uri, I thought scheme was always there. verifying that though.